Documentation/gettingstarted.rst

```diff
@@ -11,9 +11,10 @@ your machine. It is designed to take 15-30 minutes.
 If you haven't read the :ref:`intro` yet, we'd encourage you to do that first.
 
 This document includes three different guides:
- * `Getting Started Using Kubernetes`_ (Estimated time 10-15 minutes)
- * `Getting Started Using Docker`_
- * `Getting Started Using Mesos`_
+ * `Getting Started Using Kubernetes`_ with HTTP (Estimated time 10-15 minutes)
```

Comment:
I suggest getting rid of the time estimation in brackets, as it's only indicated for the first guide, and already stated in the first paragraph.

Reply:
Also, we should fix the links to include the protocol on all four lines, or not. Currently only the last GSG "with Kafka" is included in the link.